### ISSUE TITLE
0.30.2 - Logspam: Device ... is potentially a Foreign gateway

### ISSUE THREAD

**Issue by zxdavb** [zxdavb/ramses_cc#86]
This error:
```
2023-11-25 18:28:09.484 WARNING (MainThread) [ramses_tx.transport] 
 Device 18:140805 is potentially a Foreign gateway, the Active gateway is 18:006402, 
 alternatively, is it a HVAC device?, configure the known_list/block_list as required
```

**ramses_rf** no longer automatically blacklists foreign gateways, as they may actually be HVAC devices.

However, the error message should only appear once.

**Comment by zxdavb** [comment]
The error message could be: `Device ... may be a Foreign gateway`

**Comment by zxdavb** [comment]
fixed with commit on ramses_rf 33f39d80 bugfix logspam - foreign gateway
